internal/website/content/ref/sql.md

```diff
@@ -25,14 +25,15 @@ https://godoc.org/gocloud.dev/postgres.
 
 For MySQL:
 
-* [GCP Cloud SQL](https://godoc.org/gocloud.dev/mysql/cloudmysql)
-* [AWS RDS](https://godoc.org/gocloud.dev/mysql/rdsmysql)
+* [GCP Cloud SQL](https://godoc.org/gocloud.dev/mysql/gcpmysql)
```

Comment:
Not to confuse with the references below, what about 
GCP Cloud SQL for MySQL (as in https://cloud.google.com/sql/docs/mysql/)
or
Google Cloud MySQL (as in https://www.google.com/search?q=google+cloud+mysql)

Reply:
Done.